source/renderer/app/containers/staking/StakePoolsListPage.js

```diff
@@ -53,14 +53,15 @@ export default class StakePoolsListPage extends Component<Props> {
     const {
       stakePoolsRequest,
       stakePools,
-      isRanking,
       selectedDelegationWalletId,
       stake,
       fetchingStakePoolsFailed,
       recentStakePools,
       getStakePoolById,
     } = staking;
     const { all } = wallets;
+    const isLoading = !isSynced || fetchingStakePoolsFailed;
+    const isRanking = staking.isRanking && stakePoolsRequest.isExecuting;
```

Comment:
What about adding `!isLoading` here?

Reply:
@daniloprates @yakovkaravelov resolved in: https://github.com/input-output-hk/daedalus/pull/2144/commits/e682113e7e5406b6ff60a064e6719facbb677699